src/Sulu/Bundle/AdminBundle/Resources/public/scss/modules/layout.scss

```diff
@@ -28,7 +28,7 @@
     overflow: hidden;
 
     .wrapper {
-        height: 100%;
+        height: calc(100% - #{$headerHeight});
```

Comment:
Kinda hacky, as already described in https://github.com/sulu/sulu/pull/2320/files#r61098046